tests/v2/test_jsonrpc_waku.nim

```diff
@@ -259,7 +259,7 @@ procSuite "Waku v2 JSON-RPC API":
     let client = newRpcHttpClient()
     await client.connect("127.0.0.1", rpcPort)
 
-    let response = await client.get_waku_v2_store_v1_messages(@[defaultContentTopic], some(StorePagingOptions()))
+    let response = await client.get_waku_v2_store_v1_messages(@[HistoryContentFilter(contentTopic: defaultContentTopic)], some(StorePagingOptions()))
```

Comment:
What happens if we don't pass any content filter here?

Reply:
I think the underlying `store` still expects a `HistoryQuery` with populated contentFilters, but I also think changing to an `Option` will be an improvement.

Reply:
The store json rpc api was designed this way with `topics` filed which is now `contentFilters`  being mandatory. I also agree that making it optional makes sense and would also match the store protocol api.

Reply:
Opened an issue re making it optional: https://github.com/status-im/nim-waku/issues/505